Cm 10.2

zhanglt1986 opened the pull request:
for nexus s

sbrissen committed:
Add Red/Blue LED notification support

This will allow the use of the red/blue led's on the sph-d710
Uses BOARD_HAS_LED_NOTIF in boardconfig.

Change-Id: Ic217ed5b0ec673d03bb596fbde0ac725a5ca0887

xplodwild committed:
exynos4: gralloc: Remove non-trivial initializers to fix build

Change-Id: Id21a3506132b2be1ff5a606858d1be2dd897338d

Commit:
samsung: update libril for xmm6260

Change-Id: Ib5e68bd53fdf5d3db29ce09a975198320337cc52

xplodwild committed:
xmm6262: Update to new RIL version

Change-Id: I10b3d90861f5332e17ec19985efb83f8d10ae05c

Commit:
Merge "xmm6262: Update to new RIL version" into cm-10.2

intervigilium committed:
macloader: add additional SEMCOSH mac addr

Change-Id: Ib0015cb828772b1723d596442df3f7e2e298c2a2

intervigilium committed:
xmm6262: Use xmm6262 libril for xmm6360 modem as well

* They appear to expose a very similar interface

Change-Id: I6793da25244258adb905e002d05bb60061487c5e

intervigilium committed:
xmm6262: Update using 4.3 libril.cpp

Change-Id: Ib884b46448b633b862cadb0f067e3180531858ae

Commit:
Samsung: Add common class for vibrator

Change-Id: I0802cc122b606b08f58731febd8bf050d22f4c57

cm-gerrit commented:
Hi there;

This is an automated reply, to let you know CyanogenMod doesn't merge pull
requests submitted directly to our github.com repositories.

You can submit your change requests through our public review system,
located at http://review.cyanogenmod.org/

That review system is an installation of Google's Gerrit code review.
If you've never interacted with Gerrit before, we also suggest you take
a look at http://wiki.cyanogenmod.org/wiki/Howto:_Gerrit

Thanks!

On Fri, 09 Aug 2013 20:58:04 -0700 you wrote:

> for nexus s
> You can merge this Pull Request by running:
> 
>   git pull https://github.com/CyanogenMod/android_hardware_samsung cm-10.2
> 
> Or you can view, comment on it, or merge it online at:
> 
>   https://github.com/CyanogenMod/android_hardware_samsung/pull/2
> 
> -- Commit Summary --
> - exynos4: gralloc: Remove non-trivial initializers to fix build
> - samsung: update libril for xmm6260
> - xmm6262: Update to new RIL version
> - Merge &quot;xmm6262: Update to new RIL version&quot; into cm-10.2
> - macloader: add additional SEMCOSH mac addr
> - xmm6262: Update using 4.3 libril.cpp
> 
> -- File Changes --
> 
> ```
> M exynos4/hal/libgralloc_ump/gralloc_module.cpp (3)
> M macloader/macloader.cpp (3)
> M ril/xmm6260/libril/Android.mk (1)
> M ril/xmm6260/libril/ril.cpp (730)
> M ril/xmm6260/libril/ril_commands.h (8)
> M ril/xmm6260/libril/ril_event.cpp (5)
> M ril/xmm6260/libril/ril_unsol_commands.h (3)
> M ril/xmm6262/libril/Android.mk (1)
> M ril/xmm6262/libril/ril.cpp (550)
> M ril/xmm6262/libril/ril_commands.h (4)
> M ril/xmm6262/libril/ril_event.cpp (4)
> M ril/xmm6262/libril/ril_unsol_commands.h (1)
> ```
> 
> -- Patch Links --
> 
> https://github.com/CyanogenMod/android_hardware_samsung/pull/2.patch
> https://github.com/CyanogenMod/android_hardware_samsung/pull/2.diff

Commit:
Merge "xmm6262: Use xmm6262 libril for xmm6360 modem as well" into cm-10.2

Commit:
remove hw class flag

Change-Id: I8b368267efa2370eeae2c57371a7517c45856c44

xplodwild committed:
macloader: Add one more MAC prefix to murata

Change-Id: Ib00ca46608a834a6a5f1e2fe996d56ec98d1ffab

Commit:
liblights: exynos4210: fix button off value

since we're writing to the command interface the correct values are:

1 - on
2 - off

Change-Id: I4e61b0eedd1dcfba32319b26cd9e29202f66fcaa

Commit:
samsung: hardware: add support for adaptive backlight

Change-Id: I59b798e9cac2cf2c6efbe31736d3361f05c81398

Commit:
Merge "samsung: hardware: add support for adaptive backlight" into cm-10.2

Commit:
exynos4210: liblights: big cleanup

http://review.cyanogenmod.org/#/c/48922/

Change-Id: I5fa9602649c5d0e902dfc9ba9e50e8197d0dfb76

Commit:
exynos4x12: liblights: generic button backlight implementation

http://review.cyanogenmod.org/#/c/48922/

Change-Id: I929fcb4f1c2a912e026328cb09f9be527adbaea9

nardholio committed:
vibrator intensity: disable tunable framework if not supported

Change-Id: I6481b82b7ee5412234d42e39f7f5560489f14af3

Commit:
Merge changes I929fcb4f,I5fa96026 into cm-10.2

* changes:
  exynos4x12: liblights: generic button backlight implementation
  exynos4210: liblights: big cleanup

Commit:
libril: handle samsungs unsol responses

Change-Id: Ia42d9999ba0df93019a8864202f093163f8c3f41

klusark committed:
Allow herring to build s5pc110

Change-Id: I1dcd00b650c7f1cf3faa8905e0161e70fa3e57f1

humberos committed:
add exynos tablet flag

Signed-off-by: Humberto Borba <humberos@gmail.com>
Change-Id: Id2a5ce769af9b87d7fbdb415ff7869f593601dc3

invisiblek committed:
macloader: add f0:27:65 as a murata mac

Change-Id: I895dc26a71253b31a2ee8ae27feeb5feb673a9bf

rmcc committed:
gralloc: Allow any process to have multiple buffer handles

Change-Id: I670ff7e03b19c07094b506a4422f99c30f06db0e

rmcc committed:
gralloc: Allow unmapping concurrent handles

If we allow registering those buffers, we need to let them go, too

Change-Id: I6c2dc862e1e783e2b0d3c7d37d60ebe2290173fa

sbrissen committed:
allow override of cabc file via property

Note 8's cabc is implemented in mdnie instead of the panel
file like other smdk4x12 devices.

Change-Id: I9bd7c4bb5899cd323e4f6861bf7ceb61f459da05

Commit:
Added more MAC's for murata.

After finding some i9300 devices who's MAC addresses were missing from the list, i searched for all murata MAC's and thought it wise to add them.

Change-Id: I9e0db81cf2b1809b37b1660a4763db82699dd452

Flamefire committed:
Fix crash on boot

There is a race condition where the sysfs-vsync thread accesses a value
(in particular a function ptr) that may not have been set yet as this
is done externally.

Change-Id: If1afb1533344e994398b9b6f5bd3dfb125313416

cm-gerrit commented:
Hi there;

This is an automated reply, to let you know CyanogenMod doesn't merge pull
requests submitted directly to our github.com repositories.

You can submit your change requests through our public review system,
located at http://review.cyanogenmod.org/

That review system is an installation of Google's Gerrit code review.
If you've never interacted with Gerrit before, we also suggest you take
a look at http://wiki.cyanogenmod.org/wiki/Howto:_Gerrit

Thanks!
